.projenrc.ts

```diff
@@ -33,7 +33,7 @@ const project = new awscdk.AwsCdkConstructLibrary({
   copyrightPeriod: '2024-',
   defaultReleaseBranch: 'main',
   deps: [
-    '@aws-cdk/aws-lambda-python-alpha',
+    `@aws-cdk/aws-lambda-python-alpha@${CDK_VERSION}-alpha.0`,
```

Comment:
There's always this dependency, I take it? Is this documented somewhere? What happens if this graduates to non-alpha?

Reply:
This is what allows us to use the `PythonFunction` construct, so as long as we need that, we'll have this dep.

This will break if the $CDK_VERSION is updated it to a version that no longer has an alpha version of this (because it's been graduated).